### PR TITLE
Task: Code review and update on users and blog apps.

### PR DESCRIPTION
**Pull request by dev-Kevo**
Hey Danny, can you please have a review in what i did and give a feedback on the same.

### CODE REVIEW

**Comment by Dannyblazer** [comment]
I'll check it out later today

File: socialblog/config/settings/base.py
Comment (Dannyblazer) [inline socialblog/config/settings/base.py:1]: Please uncomment changes that are person to your configuration on this project it doesn't disrupt my local configuration once merge with the repo

**Review by Dannyblazer — APPROVED** [review]
I've setup the .env to hold the database creds

File: socialblog/config/settings/base.py
Comment (dev-Kevo) [reply socialblog/config/settings/base.py:1]: ok cool

**Pull request closed without merging**